templates/ldap.conf.j2

```diff
@@ -56,7 +56,7 @@
     <Group>
         BaseDN "{{ ldap.group_base_dn }}"
         SearchFilter "{{ ldap.group_search_filter }}"
-        MemberAttribute uniqueMember
```

Comment:
Make this a variable with the default set to `uniqueMember`, changing it here impacts everyone that uses the playbook.